main.cpp

```diff
@@ -62,64 +62,99 @@ send me a DM to check your pull request
 #include <string>
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
+    int value;
+    std::string name;
+
+    T(int v, const char* charPointer)
+    {
+        value = v;
+        name = charPointer;
+    }  //1
     //2
     //3
 };
 
-struct <#structName1#>                                //4
+
+struct MyStruct                                //4
 {
-    <#type name#> compare(<#type name#> a, <#type name#> b) //5
+
+    T* compare(T* a, T* b) //5
     {
-        if( a->value < b->value ) return a;
-        if( a->value > b->value ) return b;
+        if(a != nullptr && b != nullptr)
+        {
+            if( a->value < b->value ) return a;
+            if( a->value > b->value ) return b;
+            return nullptr;
+        }
+
         return nullptr;
     }
 };
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float floatOne { 0 }, floatTwo { 10 };
+    float updateValue(float* newValue)      //12
     {
-        
+        if(newValue != nullptr)
+        {
+            std::cout << "U's floatOne value: " << this->floatOne << std::endl;
+            this->floatOne = *newValue;
+            std::cout << "U's floatOne updated value: " << this->floatOne << std::endl;
+            while( std::abs(this->floatTwo - this->floatOne) > 0.001f )
+            {
+                /*
+                 write something that makes the distance between that->floatTwo and that->floatOne get smaller
+                 */
+                this->floatTwo -= 1;
+            }
+            std::cout << "U's floatTwo updated value: " << this->floatTwo << std::endl;
+            return this->floatTwo * this->floatOne;
+        }
+
+        return 0;    
     }
 };
 
-struct <#structname2#>
+struct MyStructTwo
 {
-    static <#returntype#> <#staticFunctionA#>(U* that, <#type name#>* <#updatedValue#> )        //10
+    static float updateStaticValue(U* that, float* newValue )        //10
     {
-        std::cout << "U's <#name1#> value: " << that-><#name1#> << std::endl;
-        that-><#name1#> = <#updatedValue#>;
-        std::cout << "U's <#name1#> updated value: " << that-><#name1#> << std::endl;
-        while( std::abs(that-><#name2#> - that-><#name1#>) > 0.001f )
+        if(that != nullptr)
         {
-            /*
-             write something that makes the distance between that-><#name2#> and that-><#name1#> get smaller
-             */
-            that-><#name2#> += ;
+            std::cout << "U's floatOne value: " << that->floatOne << std::endl;
+            that->floatOne = *newValue;
```

Comment:
- [ ] What if `newValue` is a nullptr??  you're using it before checking if it's safe to use.